tests/codebuild/run_canarytest_china.sh

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env bash
+
+# Env variables that need to be set before running this script
+# ROLE_ARN : SageMaker executor role arn
+# DATA_BUCKET : S3 bucket name where input data is stored
+
+if [[ -z "${ROLE_ARN}" ]]; then
+  echo "ROLE_ARN environment variable not found"
+  exit 1
+fi
+if [[ -z "${DATA_BUCKET}" ]]; then
+  echo "DATA_BUCKET environment variable not found"
+  exit 1
+fi
+
+
+# local variables
+DEPLOYMENT_NAME="ephemeral-operator-canary-"$(date '+%Y-%m-%d-%H-%M-%S')""
+CLUSTER_NAME=${DEPLOYMENT_NAME}-cluster
+CLUSTER_REGION=${CLUSTER_REGION:-cn-northwest-1}
+OIDC_ROLE_NAME=pod-role-$DEPLOYMENT_NAME
+AWS_ACC_NUM=$(aws sts get-caller-identity --region $CLUSTER_REGION   --query Account --output text)
+
+
+function create_eks_cluster() {
+  eksctl create cluster --name $CLUSTER_NAME --region $CLUSTER_REGION --auto-kubeconfig --timeout=30m --managed --node-type=c5.xlarge --nodes=1
+}
+
+function install_k8s_operators() {
+  echo installing SageMaker operators
+  aws --region $CLUSTER_REGION eks update-kubeconfig --name $CLUSTER_NAME
+  eksctl utils associate-iam-oidc-provider --cluster $CLUSTER_NAME --region $CLUSTER_REGION --approve
+
+  OIDC_URL=$(aws eks describe-cluster --region $CLUSTER_REGION --name $CLUSTER_NAME --query "cluster.identity.oidc.issuer" --output text | cut -c9-)
+
+  printf '{
+    "Version": "2012-10-17",
+    "Statement": [
+      {
+        "Effect": "Allow",
+        "Principal": {
+          "Federated": "arn:aws-cn:iam::'$AWS_ACC_NUM':oidc-provider/'$OIDC_URL'"
```

Comment:
I feel that the `aws-cn` prefix could easily be made into a variable, here, which should allow for easier integration back into the US system.

Reply:
Agreed, this is a lot of code duplication and updates will need to be made both places. 
The test yaml files should also not be copied, inject/replace the region into them.

Reply:
Yep 

Lot of other stuff need to be changed to support REGION variable 
will take this up as post release task